packages/react-components/src/DocsComponents/KnownIssues.stories.tsx

```diff
@@ -0,0 +1,147 @@
+import { makeStyles } from '../index';
+import * as React from 'react';
+import { repository as packageRepository } from '../../package.json';
+
+type GitHubUser = {
+  login: string;
+};
+
+/* eslint-disable @typescript-eslint/naming-convention */
+type GitHubIssue = {
+  html_url: string;
+  title: string;
+  number: number;
+  created_at: Date;
+  user: GitHubUser;
+};
+/* eslint-enable @typescript-eslint/naming-convention */
+
+const useStyles = makeStyles({
+  issue: {
+    display: 'flex',
+    paddingTop: '8px',
+    paddingRight: '8px',
+    paddingBottom: '8px',
+    paddingLeft: '8px',
+  },
+  openIssueIconContainer: {
+    paddingRight: '8px',
+    fill: '#1a7f37',
+    flexShrink: 0,
+    flexGrow: 0,
+  },
+  openIssueIcon: {
+    paddingTop: '8px',
+    paddingLeft: '16px',
+    fill: '#1a7f37',
+  },
+  issueTitle: {
+    fontWeight: 600,
+    fontSize: '16px',
+    color: '#24292f',
+    textDecorationLine: 'none',
+    fontFamily: '-apple-system,BlinkMacSystemFont,"Segoe UI",Helvetica,sans-serif,"Apple Color Emoji","Segoe UI Emoji"',
+  },
+  issueDetails: {
+    flexBasis: '100%',
+    fontSize: '12px',
+    marginTop: '4px',
+    color: '#57606a',
+  },
+  link: {
+    color: '#0078d4',
+    textDecorationLine: 'none',
+  },
+});
+
+export const KnownIssues: React.FC<{ componentName: string }> = ({ componentName }) => {
+  const [isError, setIsError] = React.useState(false);
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [isLoaded, setIsLoaded] = React.useState(false);
+  const [issuesList, setIssuesList] = React.useState<GitHubIssue[]>([]);
+
+  React.useEffect(() => {
+    async function fetchData() {
+      setIsLoading(true);
+      setIsError(false);
+
+      const repositoryIdentifier = packageRepository.url.split('/').slice(-2).join('/');
+
+      try {
+        const response = await fetch(
+          `https://api.github.com/repos/${repositoryIdentifier}/issues?` +
+            new URLSearchParams({
+              labels: ['Type: Bug :bug:', 'Fluent UI vNext', 'Component: ' + componentName].join(','),
+            }),
+        );
+        const issuesJson: GitHubIssue[] = await response.json();
+
+        if ((response.status >= 400 && response.status < 600) || !Array.isArray(issuesList)) {
+          throw new Error('Bad response returned from GitHub');
+        }
+
+        setIssuesList(issuesJson);
+        setIsLoaded(true);
```

Comment:
There is no reason for this `isLoaded` state for the code below. Please leverage `isLoading` only.